app/helpers/application_helper/button/general_image.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationHelper::Button::GeneralImage < ApplicationHelper::Button::Basic
```

Comment:
On naming: is _General_ something like _Basic_ here? I would stick to a consistent naming scheme